### ISSUE TITLE
The swipe function is fine, but pinch zoom is very unstable.

### ISSUE THREAD

**Issue by MaximusBaton** [antonKalinin/react-native-image-view#56]

**Comment by vedmant** [comment]
Yes, I have the same issue on android, it just doesn't work properly, it zooms in when I try to zoom out. Also expo example crashes often.

**Comment by Naishadh8115** [comment]
i am also having same issue, very unstable zoom in and zoom out.

**Comment by V1sk** [comment]
same issue here

**Comment by OlaAde** [comment]
Same issue here. Zooms in when I try to zoom out

**Comment by antonKalinin** [comment]
Hey @OlaAde, could you please check the remastered clone of this package: https://github.com/jobtoday/react-native-image-viewing. It has almost the same API and the zoom functionality should be fine there. I planing to transfer the package name and release react-native-image-view v3.0 in near future, but for now you can use https://github.com/jobtoday/react-native-image-viewing
